invenio_app_ils/config.py

```diff
@@ -913,11 +913,13 @@ def _(x):
             moi=dict(terms=dict(field="mode_of_issuance")),
             language=dict(terms=dict(field="languages")),
             relation=dict(terms=dict(field="relation_types")),
+            sertype=dict(terms=dict(field="series_type")),
```

Comment:
```suggestion
            series_type=dict(terms=dict(field="series_type")),
```

Reply:
I named it `sertype` to be similar to the `doctype` filter for documents.